website/source/api/discovery-chain.html.md

```diff
@@ -0,0 +1,541 @@
+---
+layout: api
+page_title: Discovery Chain - HTTP API
+sidebar_current: api-discovery-chain
+description: |-
+  The /discovery-chain endpoints are for interacting with the discovery chain.
+---
+
+# Discovery Chain HTTP Endpoint <sup>beta</sup>
+
+The `/discovery-chain` endpoint returns the compiled [discovery
+chain](/docs/internals/discovery-chain.html) for a service.
+
+This will fetch all related [configuration
+entries](/docs/agent/config_entries.html) and then normalize and distill them
+down into a concrete, actionable format for use by a [connect
```

Comment:
What does an actionable format look like? JSON? Can we replace distill with condense or another more commonly used term?

Reply:
How about:
```
This will fetch all related [configuration entries] and render them
into a form suitable for use by a [connect proxy] implementation.
```